lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -275,15 +275,19 @@ newDBLayer fp = do
               ExceptT $ runQuery' $
               selectWallet wid >>= \case
                   Just _ -> do
-                      let (metas, txins, txouts) = mkTxHistory wid txs
+                      let (metas, txins, txouts) = mkTxHistory wid $ W.invariant
+                              ("putTxHistory has been called with pending txs: "
+                                <> show txs)
+                              txs
+                              (not . foldl (\b a -> (b || W.isPending a)) False)
```

Comment:
Does the `any` function work here?

Reply:
Ah yes! I initially used that on a Map and then moved the code around. `any` would read better!